deskpro/deskpro-admin-frontend
# json-schema-to-yup for Dynamic settings

To test:
- Go to `/agent/settings`
- Remove value from `Agent Timeout`
- Try to submit, see error under field

Notes: Currently there is a bug in schema-to-yup library, which is fixed in their master branch. This is the branch that is being pulled in via package.json, which we'll need to change this in the future.I'm in touch with the developer to see when this fix will be tied to a versioned release. Once its available, I will update package.json.
